### ISSUE TITLE
install-node-and-yarn does not detect previously installed Yarn

### ISSUE THREAD

**Issue by wardr** [eirslett/frontend-maven-plugin#519]
When running install-node-and-yarn multiple times, it does not detect that Yarn was installed when run previously and instead installs Yarn from scratch each time. See log messages:   
```
[INFO] --- frontend-maven-plugin:1.2:install-node-and-yarn (install node and yarn) @ example ---
[INFO] Node v6.9.1 is already installed.
[INFO] Yarn 0.16.1 was installed, but we need version v0.16.1
[INFO] Installing Yarn version v0.16.1
[INFO] Unpacking C:\Users\xxx\.m2\repository\com\github\eirslett\yarn\0.16.1\yarn-0.16.1.\yarn-0.16.1.msi into C:\Users\xxx\IdeaProjects\example\build\node\yarn
[INFO] Installed Yarn locally.
```

The relevant excerpt from the POM:

```
<execution>
    <id>install node and yarn</id>
    <goals>
        <goal>install-node-and-yarn</goal>
    </goals>
    <configuration>
        <nodeVersion>v6.9.1</nodeVersion>
        <yarnVersion>v0.16.1</yarnVersion>
    </configuration>
</execution>
```

**Comment by ppenalver** [comment]
I confirm the bug in version 1.2, we have the same problem.

**Comment by schue30** [comment]
This bug has been fixed with PR#511: https://github.com/eirslett/frontend-maven-plugin/pull/511

**Comment by wardr** [comment]
Thanks; this is an awesome plugin by the way!

**Comment by schue30** [comment]
this issue can be closed. it should be fixed in release 1.3

**Comment by eirslett** [comment]
yup!
